app/uk/gov/hmrc/gatekeeper/models/xml/XmlOrganisation.scala

```diff
@@ -35,3 +37,15 @@ case class XmlOrganisation(organisationId: OrganisationId, vendorId: VendorId, n
 object XmlOrganisation {
   implicit val formatOrganisation: OFormat[XmlOrganisation] = Json.format[XmlOrganisation]
 }
+
+case class Collaborator(userId: UserId, email: LaxEmailAddress)
+
+object Collaborator {
+  implicit val formatCollaborator: OFormat[Collaborator] = Json.format[Collaborator]
+}
+
+case class XmlOrganisationWithCollaborators(organisationId: OrganisationId, vendorId: VendorId, name: String, collaborators: List[Collaborator])
```

Comment:
Is it worth just expanding the above XmlOrg to have collaborators on it? As I believe its the same object across all the endpoints in Xml Services

Reply:
Good point - done